aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -188,26 +188,15 @@ Some applications may be configured to emit multiple OpenAPI documents, for vari
 
 ### Customizing run-time behavior during build-time document generation
 
-Under the hood, build-time OpenAPI document generation functions by launching the application's entrypoint with an inert server implementation. This is a requirement to produce accurate OpenAPI documents since all information in the OpenAPI document cannot be statically analyzed. Because the application's entrypoint is invoked, any logic in the applications' startup will be invoked. This includes code that injects services into the DI container or reads from configuration. In some scenarios, it's necessary to restrict the codepaths that will run when the application's entry point is being invoked from build-time document generation. These scenarios include:
+Build-time OpenAPI document generation functions by launching the apps entrypoint with a mock server implementation. A mock server is required to produce accurate OpenAPI documents because all information in the OpenAPI document can't be statically analyzed. Because the apps entrypoint is invoked, any logic in the apps startup is invoked. This includes code that injects services into the [DI container](xref:fundamentals/dependency-injection) or reads from configuration. In some scenarios, it's necessary to restrict the code paths that will run when the apps entry point is being invoked from build-time document generation. These scenarios include:
```

Comment:
```suggestion
Build-time OpenAPI document generation functions by launching the app's entry point with a mock server implementation. A mock server is required to produce accurate OpenAPI documents because some of the information in the OpenAPI document can't be statically analyzed. Because the app's entry point is invoked, any logic in the app's startup is invoked. This includes code that injects services into the [DI container](xref:fundamentals/dependency-injection) or reads from configuration. In some scenarios, it's necessary to restrict the code paths that will run when the apps entry point is being invoked from build-time document generation. These scenarios include:
```